packages/shared/src/configs/markets.json

```diff
@@ -5,6 +5,24 @@
     "url": {
       "mainnet": "https://opensea.io/assets/",
       "testnet": "https://testnets.opensea.io/assets/"
+    },
+    "path": {
```

Comment:
This is nice, but a bit seems that does not DRY with network configs.

Reply:
agree but couldn't come up with a better way. let me merge it for now